### ISSUE TITLE
Convert the super rentals app to the Octane preview

### ISSUE THREAD

**Issue by jenweber** [ember-learn/super-rentals#118]
I chatted with @vaidehijoshi who is interested to help with this!

**Comment by vaidehijoshi** [comment]
@jenweber feel free to assign me to this (looks like i don't actually have the permissions to do that). i'll get started on it in the next week or two once i recover from emberconf 😅

**Comment by jenweber** [comment]
Hi @vaidehijoshi, just pinged you on Discord as well, but in case you don't check it much, is this issue something you still want to work on? Do you think you could have a PR in the next week or so? Let me know. Thanks!

**Comment by vaidehijoshi** [comment]
@jenweber Hi Jen! I actually _just_ started working on this issue earlier this week and started my own branch for it. But I've been pretty sick the past day or two and haven't been able to work on it since then. 😕 I am still interested and can try to get it done in another week or two? I got a little stuck on it (somehow I broke all the CSS but haven't investigated why/how) when I started upgrading earlier this week, but hopefully I can figure it out once I feel well enough to work on it again.
